web-console/knikubevirt/VMIs/VMIs.md

```diff
@@ -0,0 +1,56 @@
+# Surfacing VMIs in the UI
+
+A virtual machine instance(VMI) is the representation of a running virtual machine.
+
+## Use Cases
+
+1. Debugging -  Previously if a user was trying to resolve an issue caused by a VMI that didn't have an owner it would not be possible to discover it in the UI. Including orphaned VMIs indicates to a user that there could be an issue that they need to address.
+
+2. On Bare metal machines installed with Kubernetes or users who install Kubevirt for CNV - On these VMs there are clusters that can be scaled up and down and they create the VMIs (and not the VMs).
+
+## Mockups
+
+In order to avoid duplication and unclutter the UI, we had finalized into an option that presents VMs and VMIs in a single list view.
+We don't surface VMIs in the list if they are owned by a VM.
+
+VMIs and VMs will be distinguished between different color badges. VMI badges color are darker (#002F5D) in order to provide proper contrast for low vision users.
+
+![single list view of VMs and VMIs ](img/VMsListW_VMIsOp2.png)
```

Comment:
Note: The `HintBlock` component used in the console ( for example in the developer view => +Add page ) does not support closing option.

Reply:
ah, well I know there is active work around updating the hint pattern so we will see it eventually 🤞